### ISSUE TITLE
Bug: Workflows module - Can't change order of workflows

### ISSUE THREAD

**Issue by avenjamin** [freescout-helpdesk/freescout#2535]
Freescout: 1.8.48
Workflows module: 1.0.35

When viewing the list of workflows - they can't be re-ordered using the re-ordering drag handles.
Tested in Safari, Firefox, and Chrome on macOS

The cursor symbol changes to the indicate they should be re-orderable but attemping to drag just selects the text instead.

Safari Web Inspector shows the following errors:

```
[Warning] jQuery.Deferred exception: Can't find variable: sortable (2) (05b8c1b3fc7ed84d9b2922d2518270c936802668267.js, line 1)
(anonymous function) — 05b8c1b3fc7ed84d9b2922d2518270c936802668267.js:554:104
l — 05b8c1b3fc7ed84d9b2922d2518270c936802668267.js:1:29465
(anonymous function) — 05b8c1b3fc7ed84d9b2922d2518270c936802668267.js:1:29763
undefined

[Error] ReferenceError: Can't find variable: sortable
	(anonymous function) (05b8c1b3fc7ed84d9b2922d2518270c936802668267.js:1:31132)
```

I tried tweaking line 237 in the initWorkflows function in /var/www/html/Modules/Workflows/Public/js/module.js:

From:
```
function initWorkflows() {
    $(document).ready(function() {
        if ($('.wf-index').length) {
            var sortables = sortable('.wf-index', {
                handle: '.handle',
            });
            ...
```

To:
```
function initWorkflows() {
    $(document).ready(function() {
        if ($('.wf-index').length) {
            var sortables = $('.wf-index').sortable({
                handle: '.handle',
            });
            ...
```

But the error changed to:

```
[Error] TypeError: $('.wf-index').sortable is not a function. (In '$('.wf-index').sortable({handle:'.handle',})', '$('.wf-index').sortable' is undefined)
	(anonymous function) (05b8c1b3fc7ed84d9b2922d2518270c936803062324.js:1:31132)
```

Looks like jqueryui isn't being included.

**Comment by freescout-helpdesk** [comment]
Are you able to reproduce the issue here: https://demo.freescout.net/mailbox/1/workflows

**Comment by avenjamin** [comment]
It works fine with the two workflows in the demo system.
I can see the sortable function in the included js file.

I tried the clear cache button in Tools, but that didn't seem to help.

I've just tried searching my installation for 'sortable' using: ```grep -r 'sortable'``` and not seeing any js functions so I'm wondering if jqueryui is included in the demo system from another module?

**Comment by freescout-helpdesk** [comment]
Fixed in Workflows Module v1.0.36.
